tests.py

```diff
@@ -208,3 +208,81 @@ def test_sip_api_already_set():
             import sip
             sip.setapi("QString", 1)
             assert_raises(ImportError, __import__, "Qt")
+
+
+def test_translate_and_UnicodeUTF8_in_pyside():
+    with pyside():
+        from Qt import QtWidgets
+
+        # this does exist in PySide
+        assert QtWidgets.QApplication.UnicodeUTF8 is not None
+
+        # use patched method with old arguments
+        QtWidgets.QApplication.translate(
```

Comment:
Ah ok. No problem.

As tests now require all bindings, I'd imagine it can get tricky to replicate this environment locally. The developer guide has instructions on how to run these via Docker, if you have access to this. It'll let you run tests locally without installing any bindings, exactly the way Travis is running it.